dotrix_core/src/renderer.rs

```diff
@@ -277,10 +279,39 @@ pub fn world_renderer(
     }
 
     // Prepare lights
-    let query = world.query::<(&mut Light,)>();
     let mut lights = LightUniform::default();
-    for (light,) in query {
-        lights.push(*light);
+
+    let query = world.query::<(&mut AmbientLight,)>();
```

Comment:
it probably should not be mutable, you only read it, don't you?

Reply:
yes, you're right